plugin/core/settings.py

```diff
@@ -85,6 +86,14 @@ def update_configs(self) -> None:
     def _set_enabled(self, config_name: str, is_enabled: bool) -> None:
         settings = sublime.load_settings("LSP.sublime-settings")
         clients = settings.get("clients")
+
+        plugin = get_plugin(config_name)
+        if plugin:
+            plugin_settings, plugin_settings_name = plugin.configuration()
+            plugin_settings.set("enabled", is_enabled)
+            sublime.save_settings(plugin_settings_name)
+            return
```

Comment:
This can be the first thing in this function as it doesn't use variables above.

Also, we tend to avoid unnecessary blank lines.